hubstorage/client.py

```diff
@@ -162,6 +170,16 @@ def close(self, timeout=None):
         if self._batchuploader is not None:
             self.batchuploader.close(timeout)
 
+    @property
+    def USERAGENT(self):
+        warnings.warn(USERAGENT_DEPRECATED_MSG, DeprecationWarning)
```

Comment:
`warnings` is not imported, I have `NameError: global name 'warnings' is not defined` locally and wonder how tests passed